### PR TITLE
java: add graphviz to container image

### PR DESCRIPTION
**Pull request by averikitsch**
For sample https://github.com/GoogleCloudPlatform/java-docs-samples/pull/1610
Reference node PR #31

### CODE REVIEW

File: java/java11/Dockerfile [hunk 1]
```diff
@@ -19,6 +19,14 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends libxml2-utils && \
     rm -rf /var/cache/apt
 
+# Add Graphviz
+RUN set -ex; \
```
Comment (chingor13) [inline java/java11/Dockerfile]: We should be able to accomplish this behavior using `&&` between the commands to match the style of the other docker build steps

**Comment by averikitsch** [comment]
@grayside any reason you used `;` instead of `&&`?

File: java/java11/Dockerfile [hunk 2]
```diff
@@ -19,6 +19,12 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends libxml2-utils && \
     rm -rf /var/cache/apt
 
+# Add Graphviz
+RUN set -ex && \
```
Comment (chingor13) [inline java/java11/Dockerfile]: with the `&&` we don't need the `set -ex`

**Comment by grayside** [comment]
Not sure how I missed this notification, sorry. Since I was asked I'll go ahead and fill in my answer.

The change in this PR ended up on:
```
RUN apt-get update -y && \
    apt-get install -y graphviz && \
    rm -rf /var/lib/apt/lists/*
```

However, I generally recommend (and use) something more like this (from the [node-12 image](https://github.com/googleapis/testing-infra-docker/blob/master/node/12-user/Dockerfile#L18))

```
RUN set -ex; \
  apt-get update -y; \
  apt-get install -y \
    graphviz \
    ; \
  rm -rf /var/lib/apt/lists/*
```

Reasons I do this:

* Semicolons are less visually intrusive
* set -e preserves the "stop-on-error" behavior of `&&`
* set -x outputs the executed commands during build, removing some of the mystery when reading build logs
* One package per line makes diffs easier to read, especially once you get more than ~5 packages.

**Pull request merged**